### ISSUE TITLE
training custom dataset problem

### ISSUE THREAD

**Issue by tdn670000** [zylo117/Yet-Another-EfficientDet-Pytorch#109]
I'm trying to detect car plate number(1 class), but cannot get good quality
using repo version of 14.04.2020 16:13 utc

I compared formats of dataloader outputs with your tutorial, its ok
set force_input_size = None
current training status:
Val. Epoch: 186/500. Classification loss: 0.54055. Regression loss: 0.05754. Total loss: 0.59809
Step: 34772. Epoch: 187/500. Iteration: 178/185. Cls loss: 0.36639. Reg loss: 0.04083. Total loss: 0.40722: 96%
178/185 [04:41<00:11, 1.59s/it]

learning rate = 1e-4
total loss isn't decreasing about last 30 epochs. and not increasing, so it's not look like it ovefitting

prediction(max confidence of bbox = 0.092):
![img_inferred_d0_this_repo_0](https://user-images.githubusercontent.com/31201238/79435441-568d8480-7fd8-11ea-8293-f71abcdd6817.jpg)

what can i do?
what value of loss is good?

**Comment by zylo117** [comment]
overfitting.

try `--head_only True`

**Comment by tdn670000** [comment]
I trained with head_only True

I understood, it was underfitting
So, I trained with "lr"=1e-3 and "head_only"=True and "load_weights" d0 from repo first
then lr=1e-3 and head_only False
then lr=1e-4

and it's result(confidence=0.974):
![12595119_0](https://user-images.githubusercontent.com/31201238/79546220-b9475480-809a-11ea-8352-8eab7440ae5e.jpg)
Thank you

**Comment by zylo117** [comment]
good job!

**Comment by xkp793003821** [comment]
@tdn670000 
good job,
can you share the final class_loss and reg_loss in your experiment when you get a good test result
thanks

**Comment by tdn670000** [comment]
![loss](https://user-images.githubusercontent.com/31201238/79694210-99b95300-8277-11ea-90fc-5e305a7dcde9.png)
But I have a new problem, my model cannot predict small objects, so I think, classification loss should be less

**Comment by zylo117** [comment]
@tdn670000 
yes, the weights are overfitting in the final 50k steps. you should fix it and then test again.

**Comment by rm2886** [comment]
@tdn670000 @zylo117 can you please tell me how to view tensorboard page.
I tried but always get a white page.


Thanks

**Comment by zylo117** [comment]
`tensorboard --logdir /path/to/the/tensorboard/log/folder/`

**Comment by xkp793003821** [comment]
@tdn670000 @zylo117 
Thanks for your work, I have get a good detection result
ahh

**Comment by mytk2012** [comment]
> I trained with head_only True
> 
> I understood, it was underfitting
> So, I trained with "lr"=1e-3 and "head_only"=True and "load_weights" d0 from repo first
> then lr=1e-3 and head_only False
> then lr=1e-4
> 
> and it's result(confidence=0.974):
> ![12595119_0](https://user-images.githubusercontent.com/31201238/79546220-b9475480-809a-11ea-8352-8eab7440ae5e.jpg)
> Thank you

Can you tell me how to visual my results except loss？I am not sure if my test results are correct
